### DISCUSSION TITLE
Category and Subcategory names added to the Catalog API

### DISCUSSION THREAD

**Discussion by DanielaZamoraUribe** [amzn/selling-partner-api-models#624] [Ideas]
The catalog API allows us to retrieve the Category and Subcategory codes of an ASIN, but not its names. For example: One ASIN can belong to the category '30400 Food Preparation', but the category I get from the API is a different number that does not match with 30400 and does not include the name "Food Preparation" either. 

It would be super helpful for us to have this data included in the API as we use these two data points to map our internal categories. Thanks! 🙏

**Comment by ANarayanan-Cervello** [comment]
@DanielaZamoraUribe I am facing the same issue . When I raised a case with Amazon team, their response was that as far as Amazon is concerned, Category and Subcategory are always numbers(2000,3000,etc) .The description corresponding to these numbers are nothing but vendor interpretation of the same.

**Comment by chapmanjw** [comment]
The value in the `attributes` data is the actual value for those attributes (the numeric values). Additionally, the `vendorDetails` section contains the numeric values in the v2020-12-01 version. We are expanding the `vendorDetails` to include the display names for these. See https://github.com/amzn/selling-partner-api-docs/discussions/2017 for a preview of those changes.

Additionally, the JSON Schema for the product types in the Product Type Definitions API includes both the values and display names of the categories and subcategories for a given product type.
